frontend/src/main/java/com/java/academy/gym/controller/InternationalizationController.java

```diff
@@ -0,0 +1,30 @@
+package com.java.academy.gym.controller;
+
+import com.java.academy.gym.dto.LocaleUserMessagesRequestDto;
+import com.java.academy.gym.service.InternationalizationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping(value = "/i18n")
+public class InternationalizationController {
+
+    private InternationalizationService internationalizationService;
+
+    @Autowired
+    public InternationalizationController(InternationalizationService internationalizationService) {
+        this.internationalizationService = internationalizationService;
+    }
+
+    @PostMapping
```

Comment:
Why POST request? I think this one is idempotent.

Reply:
@mprtcz I think you mean that POST is not idempotent.
We had discussion with @arturAR about that and in the first version I implemented this as GET request with params in URL. @arturAR wanted to have POST with params in @RequestBody.
I don't know what approach should be taken here, because in GET request we should't have body (https://stackoverflow.com/questions/978061/http-get-with-request-body etc.). So maybe you can ask @volodymyrd what is correct approach in such situations? For me POST is also weird here...

Reply:
No, I meant that this method is. And POST is not. So POST should definitely not go here.
[There yo ugo](https://evertpot.com/dropbox-post-api/)
Better to use PATCH or PUT as those methods are idempotent and can have request body. They are not safe, but we will not change anything with this method so it would suffice.
Oh, and apparently GET can have body after all :suspect:

Reply:
GET here is more suitable, but to be honest, we shouldn't spent a lot of time discussing it, it's just theory, I prefer use GET only for read-only operations for the rest one I use POST, because using different methods makes your API more complicated to use and support, and I'm not sure about benefits.